sumdbaudit/README.md

```diff
@@ -63,6 +65,60 @@ connection, and in around 10 minutes on a Raspberry Pi connected over WiFi.
 Your mileage may vary. At the time of this commit, SumDB contained a little over
 1.5M entries which results in a SQLite file of around 650MB.
 
+## Setting up a `mirror` service
+These instructions show how to set up a mirror service to run on a Raspberry Pi
+running a recent version of Raspbian.
+
+> :frog: this would be more useful with a client/server database instead of sqlite!
+
+Setup:
+```bash
+# Build the mirror and install it where it can be executed
+go build ./sumdbaudit/cli/mirror
+sudo mv mirror /usr/local/bin/sumdbmirror
+
+# Create a user to run the service that has no login
+sudo useradd -M sumdb
+sudo usermod -L usermod -s /bin/false sumdb
+
+# Create a directory to store the sqlite database
+sudo mkdir /var/cache/sumdb
+sudo chown sumdb.sumdb /var/cache/sumdb
+```
+Define the service by creating the file `/etc/systemd/system/sumdbmirror.service` with contents:
+```
+[Unit]
+Description=Go SumDB Mirror
+After=network.target
+[Service]
+Type=simple
+User=sumdb
+ExecStart=/usr/local/bin/sumdbmirror -db /var/cache/sumdb/mirror.db -alsologtostderr -v=1
+
+[Install]
+WantedBy=multi-user.target
+```
+
+Start the service and check its progress:
+```bash
+sudo systemctl daemon-reload
+sudo systemctl start sumdbmirror
+
+# Follow the latest log messages
+journalctl -u sumdbmirror -f
+```
+
+When the mirror service is sleeping, you will be able to query the local database at
+`/var/cache/sumdb/mirror.db` using the example queries in the next section.
+At the time of writing this setup uses almost 600MB of storage for the database.
+
+If you want to have the `leafMetadata` table populated then you can add an extra argument
+to the service definition.
+In the `ExecStart` line above, add `-unpack` and then restart the `sumdbmirror` service.
```

Comment:
Maybe add the systemctl command to restart the service here too?

Reply:
Yeah, good call.